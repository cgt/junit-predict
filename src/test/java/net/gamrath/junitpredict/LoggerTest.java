package net.gamrath.junitpredict;

import org.approvaltests.Approvals;
import org.junit.jupiter.api.Test;

import java.util.List;

class LoggerTest {
    @Test
    void name() {
        final var log = Logger.createLog(List.of(), false, Prediction.ALL_PASS);
        Approvals.verify(String.join("\n", log) + "\n");
    }
}