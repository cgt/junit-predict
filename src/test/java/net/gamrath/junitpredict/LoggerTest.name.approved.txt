ALL_PASS,false
STATS: hits=0, misses=1
